python/private/render_pkg_aliases.bzl

```diff
@@ -56,27 +57,27 @@ def _render_whl_library_alias(
     # whls that are based on a specific version of Python.
     selects = {}
     no_match_error = "_NO_MATCH_ERROR"
-    default = None
     for alias in sorted(aliases, key = lambda x: x.version):
         actual = "@{repo}//:{name}".format(repo = alias.repo, name = name)
-        selects[alias.config_setting] = actual
+        selects.setdefault(actual, []).append(alias.config_setting)
         if alias.version == default_version:
-            default = actual
+            selects[actual].append("//conditions:default")
             no_match_error = None
 
-    if default:
-        selects["//conditions:default"] = default
-
     return render.alias(
         name = name,
         actual = render.select(
-            selects,
+            {tuple(sorted(v, key = lambda x: ("is_python" not in x, x))): k for k, v in sorted(selects.items())},
```

Comment:
Adding a comment here would be super helpful. It takes a bit of time to understand what this line done.